includes/rest-api.php

```diff
@@ -379,7 +379,7 @@ function get_pull_content_permissions( $request ) {
 		&& null !== $request->get_param( 'distributor_request' )
 		&& (
 			null === $request->get_header( 'X-Distributor-Version' )
-			|| version_compare( $request->get_header( 'X-Distributor-Version' ), '2.0.0', '<' )
+			|| version_compare( $request->get_header( 'X-Distributor-Version' ), '2.0.0-alpha', '<' )
```

Comment:
`version_compare` treats `2.0.0-beta1` as prior to `2.0.0` so this line needs to change to account for tagging a pre-release version of the software.